contrib/test/ci/vars.yml

```diff
@@ -129,6 +129,7 @@ kata_skip_ctr_tests:
   - 'test "annotations passed through"'
   - 'test "ctr with default_env set in configuration"'
   - 'test "ctr with absent mount that should be rejected"'
+  - 'test "ctr stop loop kill retry attempts"'
```

Comment:
**To the reviewer:**

The Runtime VM (Kata) implements its own `StopContainer()` with custom logic, log lines, etc. Hence, the OCI test does not apply to it.